bootstrap.sh

```diff
@@ -3,7 +3,7 @@
 # fails while the -x option makes verbosely it output what it does
 
 # Install Pipenv, the -n option makes sudo fail instead of asking for a
-# password if we don�t have sufficient privileges to run it
+# password if we don�t have sufficient privileges to run it
```

Comment:
Something's off with this encoding, please figure out how to make this a normal apostrophe; I would also suggest trying to check the rest of your edits for weird encoding issues. If you (accidentally) put a Hebrew apostrophe here - then that's a big no-no, but I'm sure that you know that already.

Reply:
Yeah, thanks. don't know how it happened. Fixed.